src/main/java/duke/PrintMethod.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+import duke.task.Task;
+import duke.task.TaskArray;
+
+import java.util.ArrayList;
+
+import java.io.IOException;
+
+public class PrintMethod {
+
+    public static final char sadFace = '\u2639';
+
+
+    public static void printLines() {
+        System.out.println("-------------------------------------------------------------");
```

Comment:
The name of the method is "printLines". However, the method only prints one line of the dash.